src/AutosizeInput.js

```diff
@@ -152,7 +158,28 @@ const AutosizeInput = createClass({
 				}
 			</div>
 		);
-	},
-});
+	};
+}
+
+AutosizeInput.propTypes = {
```

Comment:
Yeah i don't imagine there are any performance improvements from using the `static` syntax